packages/web3-eth-contract/test/integration/conract_deploy.test.ts

```diff
@@ -132,7 +132,7 @@ describe('contract', () => {
 			).rejects.toThrow('Returned error: intrinsic gas too low');
 		});
 
-		it('should fail with errors deploying a zero length bytecode', () => {
+		it.skip('should fail with errors deploying a zero length bytecode', () => {
```

Comment:
Why do you skip this test?

Reply:
Yes I fixed this one in my recent PR. This should not be skipped.

Reply:
@nazarhussain Can I just remove the `skip`? This branch is up to date with `4.x`, so not sure how this snuck in here

Reply:
@spacesailor24 Yes you can remove the skip.

Reply:
It's been removed